README.md

```diff
@@ -18,7 +18,7 @@ Sarama is an MIT-licensed Go client library for [Apache Kafka](https://kafka.apa
 Sarama provides a "2 releases + 2 months" compatibility guarantee: we support
 the two latest stable releases of Kafka and Go, and we provide a two month
 grace period for older releases. This means we currently officially support
-Go 1.4 and 1.5, and Kafka 0.8.1 and 0.8.2, although older releases are still
+Go 1.4 and 1.5, and Kafka 0.8.2 and 0.9.0, although older releases are still
```

Comment:
Pedantic: I think we should mention the most recent version first, and then the older version(s) we still support, for both go and kafka.